From 69e5fd82bf5d9282a640ae8f4069e3cd1bf6abb8 Mon Sep 17 00:00:00 2001
From: Pavel Grunt <pgrunt@redhat.com>
Date: Thu, 23 Mar 2017 12:26:28 +0100
Subject: [PATCH 23/55] vdagentd: Handle VD_AGENT_CLIPBOARD_RELEASE

In fact do nothing but avoid a runtime warning
 vdagent_message_clipboard_from_le: code should not be reached

Introduced in 99d9d3583143aef7143ec986cebe2980fdeeb776

Acked-by: Victor Toso <victortoso@redhat.com>
---
 src/vdagentd/vdagentd.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/vdagentd/vdagentd.c b/src/vdagentd/vdagentd.c
index 954bba1..aef5b96 100644
--- a/src/vdagentd/vdagentd.c
+++ b/src/vdagentd/vdagentd.c
@@ -410,6 +410,8 @@ static void vdagent_message_clipboard_from_le(VDAgentMessage *message_header,
     case VD_AGENT_CLIPBOARD_GRAB:
         virtio_msg_uint32_from_le(data, message_header->size, min_size);
         break;
+    case VD_AGENT_CLIPBOARD_RELEASE:
+        break;
     default:
         g_warn_if_reached();
     }
-- 
2.16.1

